lib/stripe/api_operations/list.rb

```diff
@@ -11,7 +11,7 @@ def list(filters = {}, opts = {})
 
         # set filters so that we can fetch the same limit, expansions, and
         # predicates when accessing the next and previous pages
-        obj.filters = filters.dup
+        obj.filters = filters.dup unless filters.nil?
```

Comment:
It's possible for filters to be `nil` due to how we delegate methods to the API resources.